packages/waku/src/lib/plugins/vite-plugin-deploy-cloudflare.ts

```diff
@@ -208,14 +209,18 @@ export default {
         force: true,
       });
 
+      unstable_writeBuildConfig(
+        path.join(outDir, WORKER_JS_NAME, DIST_ENTRIES_JS),
+      );
```

Comment:
I don't like the complexity and exposing the function from build.ts (which doesn't work if you own the plugin outside waku repo.)

Let's simply do this here:
```suggestion
      appendFileSync(
        path.join(outDir, WORKER_JS_NAME, DIST_ENTRIES_JS),
        `export const buildData = ${JSON.stringify(platformObject.buildData)};`,
      );
```

We will revisit the build process in the future anyway.

Reply:
OK. A library of helper functions for deploy authors to use would be nice. The "isNodeCompatible" replacement is another candidate for a helper function:

```ts
      config.ssr.target = 'webworker';
      config.ssr.resolve ||= {};
      config.ssr.resolve.conditions ||= [];
      config.ssr.resolve.conditions.push('worker');
      config.ssr.resolve.externalConditions ||= [];
      config.ssr.resolve.externalConditions.push('worker');
```